configs/grafana/grafana.ini

```diff
@@ -1,1155 +0,0 @@
-;viewers_can_edit = false
-
-# Editors can administrate dashboard, folders and teams they create
-;editors_can_admin = false
-
-# The duration in time a user invitation remains valid before expiring. This setting should be expressed as a duration. Examples: 6h (hours), 2d (days), 1w (week). Default is 24h (24 hours). The minimum supported duration is 15m (15 minutes).
-;user_invite_max_lifetime_duration = 24h
-
-# Enter a comma-separated list of users login to hide them in the Grafana UI. These users are shown to Grafana admins and themselves.
-; hidden_users =
-
-[auth]
-# Login cookie name
-;login_cookie_name = grafana_session
-
-# The maximum lifetime (duration) an authenticated user can be inactive before being required to login at next visit. Default is 7 days (7d). This setting should be expressed as a duration, e.g. 5m (minutes), 6h (hours), 10d (days), 2w (weeks), 1M (month). The lifetime resets at each successful token rotation.
-;login_maximum_inactive_lifetime_duration =
-
-# The maximum lifetime (duration) an authenticated user can be logged in since login time before being required to login. Default is 30 days (30d). This setting should be expressed as a duration, e.g. 5m (minutes), 6h (hours), 10d (days), 2w (weeks), 1M (month).
-;login_maximum_lifetime_duration =
-
-# How often should auth tokens be rotated for authenticated users when being active. The default is each 10 minutes.
-;token_rotation_interval_minutes = 10
-
-# Set to true to disable (hide) the login form, useful if you use OAuth, defaults to false
-disable_login_form = true
-
-# Set to true to disable the sign out link in the side menu. Useful if you use auth.proxy or auth.jwt, defaults to false
-;disable_signout_menu = false
-
-# URL to redirect the user to after sign out
-;signout_redirect_url =
-
-# Set to true to attempt login with OAuth automatically, skipping the login screen.
-# This setting is ignored if multiple OAuth providers are configured.
-;oauth_auto_login = false
-
-# OAuth state max age cookie duration in seconds. Defaults to 600 seconds.
-;oauth_state_cookie_max_age = 600
-
-# Skip forced assignment of OrgID 1 or 'auto_assign_org_id' for social logins
-;oauth_skip_org_role_update_sync = false
-
-# limit of api_key seconds to live before expiration
-;api_key_max_seconds_to_live = -1
-
-# Set to true to enable SigV4 authentication option for HTTP-based datasources.
-;sigv4_auth_enabled = false
-
-# Set to true to enable verbose logging of SigV4 request signing
-;sigv4_verbose_logging = false
-
-#################################### Anonymous Auth ######################
-[auth.anonymous]
-# enable anonymous access
-enabled = true
-
-# specify organization name that should be used for unauthenticated users
-org_name = Main Org.
-
-# specify role for unauthenticated users
-org_role = Admin
-
-# mask the Grafana version number for unauthenticated users
-hide_version = true
```

Comment:
@mairp I see that you removed this file, which is likely causing the login form to appear?
I haven't used this ini file myself, but I tried a few things in srl-telemetry-labs to disable the login form and allow anonymous users to get access to dashboards, but I failed to find the working solution.
Have you succeeded? If yes, maybe we need to keep a working solution?

Reply:
@hellt Indeed this how I was able to enable anonymous access on release 8.5.2, it works fine! I need to check what things have been changed in the 9.5.2 in regards this .ini file and see if it would be something similar or not. I can give a shot!

Reply:
yes, it worked for me in the latest grafana v8.5.25 but doesn't in v9.
I wonder what's changed...